test/relayer.js

```diff
@@ -0,0 +1,92 @@
+const { formatBytes32String } = require("ethers").utils;
+
+const Wallet = require("../build/BaseWallet");
+const TestModule = require("../build/TestModule");
+const TestModuleRelayerV2 = require("../build/TestModuleRelayerV2");
+const Registry = require("../build/ModuleRegistry");
+
+const TestManager = require("../utils/test-manager");
+const { getRandomAddress } = require("../utils/utilities.js");
+
+
+const TARGET_OF_DATA_NOT_WALLET_REVERT_MSG = "RM: the wallet authorized is different then the target of the relayed data";
+const TARGET_OF_DATA_NOT_WALLET_REVERT_MSG_V2 = "RM: Target of _data != _wallet";
+const INVALID_DATA_REVERT_MSG = "RM: Invalid dataWallet";
+const DUPLICATE_REQUEST_REVERT_MSG = "RM: Duplicate request";
+
+describe("Test RelayerModule", function () {
```

Comment:
Kudos for adding a dedicated set of tests for the `RelayerModule` 💯

Reply:
All tests are called just by their name, rather than prefixed with "Test". For consistency can we rename to just "RelayerModule"

Reply:
Ok, I renamed all test descriptions that were prefixed with "Test ".